model_compression_toolkit/core/common/quantization/quantization_params_generation/error_functions.py

```diff
@@ -349,6 +381,7 @@ def get_threshold_selection_tensor_error_function(quantization_method: Quantizat
         quantization_method: Quantization method for threshold selection
```

Comment:
quantization_method: Method used for selecting the quantization threshold.
quant_error_method: Type of error function requested.
p: P-norm to use for calculating the Lp-norm distance.
axis: Axis along which the operation has been performed.
norm: Indicates whether to normalize the result of the error function.
n_bits: Number of bits used to quantize the tensor.
signed: Indicates whether the input is signed.